docs/manual/manual_stock_led.rst

```diff
@@ -0,0 +1,251 @@
+You can find RGB LED controls on the General tab. Note that the values seen in the image below are those required to create the light blue, default, color
+
+.. image:: ../_static/manual_images/leds/rgb_led_cc_params.png
+
+To configure your LED to be green, simply set ``RGB LED Blue`` and ``RGB LED Red`` to 0, and set ``RGB LED Update Color``. You can adjust the intensity of the green LED 
+as desired, making sure to set ``RGB LED Update Color`` in order to test the configured intensity. In this example, we set a green intensity of 100. 
+Note the color and intensity of the LED.
+
+.. image:: ../_static/manual_images/leds/rgb_led_cc_green.png
+
+Since every set through the Control Center also saves the value into the module's persistent memory, this RGB configuration will be active on every module startup 
+until the value is changed and saved again. So, if you restart your module, you will find that it now starts with the same color and intensity as configured 
+through the Control Center.
+
+An important note is that it is not required to set ``RGB LED Update Color`` in order to have your module boot with the previously saved color. The module will 
+always boot with the value loaded from persistent memory. Suppose you want your module to default to magenta. We can follow the same steps as above with 
+IQ Control Center to achieve this, but this example will use the Python API. Note that we do not call ``module.set("rgb_led", "update_color")``, but the 
+module reboots with the value set and saved by the program.
+
+.. code-block:: python
+
+    import iqmotion as iq
+
+    # Module Communication - This must be updated with the correct parameters for your module's serial communication
+    com = iq.SerialCommunicator("COM3", baudrate=115200)
+
+    # Using a Vertiq 40-06 Gen 2 module with speed firmware
+    module = iq.Vertiq4006(com, firmware="speed")
+
+    #Set and save the RGB LED to magenta (red and blue only)
+    module.set("rgb_led", "red", 100)
+    module.save("rgb_led", "red")
+    module.set("rgb_led", "green", 0)
+    module.save("rgb_led", "green")
+    module.set("rgb_led", "blue", 255)
+    module.save("rgb_led", "blue")
+
+    #Reboot the module to show that it will boot to the color we just set
+    module.set("system_control", "reboot_program")
+
+White LED Configuration
+==========================
+Your module has control over the two series white LEDs' intensity as well as their strobing output. Strobing configuration is identical for both the RGB and white LEDs, 
+and is covered in :ref:`strobing_configuration`. By default, the white LED intensity is 0 and strobing is enabled. White LED control parameters are 
+found in the :ref:`white LED message table <white_led_message_table>`.  The LED intensity can be an integer [0, 100] where 100 indicates a maximum intensity. Please note that the 
```

Comment:
Maybe worth clarifying that this is 0 to 100 is meant to represent a percentage of intensity. Maybe obvious, but it may streamline some of this description if you can just say that you are setting it from 0% to 100% intensity.

The percentage is something you could carry through to a lot of the white discussion throughout here